app/routes/bdb/components/BdbPage.tsx

```diff
@@ -148,9 +155,40 @@ const BdbPage = () => {
         Du kan endre semesterstatuser ved å trykke på dem i listen!
       </Card>
 
+      <SelectInput
+        name="semester"
+        options={companySemesters
+          .sort((a, b) => (b.semester === 'autumn' ? 1 : -1))
+          .sort((a, b) => b.year - a.year)
+          .map((semester) => ({
+            label: semesterToHumanReadable(
+              semester as TransformedSemesterStatus,
+            ),
+            value: semester.id as number,
+          }))}
+        value={{
+          label: currentCompanySemester
+            ? semesterToHumanReadable(
+                currentCompanySemester as TransformedSemesterStatus,
+              )
+            : 'Velg semester',
+          value: currentCompanySemester?.id as number,
+        }}
+        onChange={(e) =>
+          setQuery({
+            ...query,
+            semester: getSemesterSlugById(
+              (e as { label: string; value: number }).value,
```

Comment:
Is there a type for this?

Reply:
I don't think so as those fields are what the `<SelectInput>`'s generic `Option` extends